Cargo.toml

```diff
@@ -112,31 +112,28 @@ bindgen = { version = "0.72.0" }
 cargo_metadata = { version = "0.18.1" }
 cbindgen = { version = "0.27.0" }
 cc = { version = "1.0.98" }
-cdr = { version = "0.2.4" }
 clap = { version = "4.5.4", features = ["derive"] }
 enum-iterator = { version = "2.1.0" }
 better-panic = { version = "0.3.0" }
 colored = { version = "2.1" }
 generic-tests = { version = "0.1.2" }
 human-panic = { version = "2.0.2" }
-lazy_static = { version = "1.4.0" }
+lazy_static = { version = "1.4.0", default-features = false }
 libc = { version = "0.2.169", features = ['extra_traits'] }
 log = { version = "0.4.21" }
-once_cell = { version = "1.19.0" }
```

Comment:
Removed this in favor of `lazy_static` to reduce the number of dependencies. Besides `Lazy`, the other components are of `once_cell` are now in `std`/`core` anyway.